### DISCUSSION TITLE
How to change git email address?

### DISCUSSION THREAD

**Discussion by radesavkovic** [community/community#61085] [Discussions] [answered]
### Select Topic Area

Question

### Body

I want to change username and email for my git commit.
After changed auth info, it doesn't work for the auth info.
How to change username and email?

**Comment by Blockchain1207** [comment, marked as answer]
I see what you mean.
You can do it with git command.

#git config user.email "YOUR EMAIL"
#git config user.name "YOUR NAME"

**Comment by radesavkovic** [comment]
Thanks for your quick answer.
